freqtrade/commands/list_commands.py

```diff
@@ -54,6 +54,26 @@ def start_list_strategies(args: Dict[str, Any]) -> None:
         print(tabulate(strats_to_print, headers='keys', tablefmt='pipe'))
 
 
+def start_list_hyperopts(args: Dict[str, Any]) -> None:
+    """
+    Print files with HyperOpt custom classes available in the directory
+    """
+    from freqtrade.resolvers.hyperopt_resolver import HyperOptResolver
```

Comment:
> Pls verify I did not add an unsolicited dependency on hyperopt at the global level.

That would only happen if you add this import to the top of the file ... which you did not, so that's fine :+1: